### PR TITLE
Title change event with data 

### PR DESCRIPTION
**Pull request by vinayak-dadhich**
Title of LWC component changed

### CODE REVIEW

**Review by Knightly-Spartan — CHANGES_REQUESTED** [review]
Add a comment explaining the change

**Review by Knightly-Spartan — APPROVED** [review]

File: force-app/main/default/lwc/eventWithData/eventWithData.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <lightning-card title="Event Data Parent and Child" icon-name="standard:logging">
```
Comment (Knightly-Spartan) [inline force-app/main/default/lwc/eventWithData/eventWithData.html:2]: You left a typo in the title, please correct that.

**Review by Knightly-Spartan — CHANGES_REQUESTED** [review]

File: force-app/main/default/lwc/eventWithData/eventWithData.html (hunk 1 above)
Comment (vinayak-dadhich) [reply force-app/main/default/lwc/eventWithData/eventWithData.html:2]: DONE

**Review by Knightly-Spartan — APPROVED** [review]

File: force-app/main/default/lwc/eventWithData/eventWithData.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+    <lightning-card title="Event Parentt and Child" icon-name="standard:logging">
         <template lwc:if={contacts.data}>
```
Comment (Knightly-Spartan) [inline force-app/main/default/lwc/eventWithData/eventWithData.html:3]: amit

Comment (vinayak-dadhich) [reply force-app/main/default/lwc/eventWithData/eventWithData.html:3]: DONE

**Pull request merged**